keyboards/gray_studio/hb85/hb85.c

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2017 Luiz Ribeiro <luizribeiro@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <string.h>
+
+#include "rgblight.h"
+
+#include "i2c.h"
+#include "quantum.h"
+
+#ifdef RGBLIGHT_ENABLE
+extern rgblight_config_t rgblight_config;
+
+void rgblight_set(void) {
+    if (!rgblight_config.enable) {
+        for (uint8_t i = 0; i < RGBLED_NUM; i++) {
+            led[i].r = 0;
+            led[i].g = 0;
+            led[i].b = 0;
+        }
+    }
+
+    i2c_init();
+    i2c_send(0xb0, (uint8_t*)led, 3 * RGBLED_NUM);
+}
+#endif
+
+__attribute__ ((weak))
+void matrix_scan_user(void) {
+}
+
+void backlight_init_ports(void) {
+	DDRD |= (1<<0 | 1<<1 | 1<<4 | 1<<6);
```

Comment:
Could you update these to use the GPIO Control functions, instead? 

https://docs.qmk.fm/#/internals_gpio_control

Reply:
@drashna can you point me in the right direction of how the GPIO functions would translate here? I will implement it on the other part myself, just need some guidance.

Reply:
Well, the actual code to handle this is here:
https://github.com/qmk/qmk_firmware/blob/master/quantum/quantum.h#L147-L169

That would be the best reference, but not the most intuitive. 

A good example is one of my commits, converting everything to these:
https://github.com/qmk/qmk_firmware/commit/40383089d035f69101851831f756508271fff103?w=1

Though, this may take a bit of testing to make sure that it's using the correct commands.